src/components/Input/TextField/styles.ts

```diff
@@ -28,6 +61,11 @@ export const styInputBorder = css({
     backgroundColor: 'white',
 });
 
+const borders = {
```

Comment:
ini kan constants ya
```suggestion
const BORDERS = {
```